modules/database/src/interfaces/ICms.ts

```diff
@@ -0,0 +1,44 @@
+interface CrudOperation {
+  enabled: boolean;
+  authenticated: boolean;
+}
+
+const crudOperationDefaults: CrudOperation = {
+  enabled: false,
+  authenticated: false,
+};
+
+export interface ICms {
+  authentication: false;
+  enabled: true;
+
+  crudOperations: {
+    create: typeof crudOperationDefaults;
+    read: typeof crudOperationDefaults;
+    update: typeof crudOperationDefaults;
+    delete: typeof crudOperationDefaults;
+  };
+}
+
+export const cmsInstance: ICms = {
```

Comment:
Rename this to `introspectedSchemaCmsOptionsDefaults`, `defaultIntrospectedSchemaCmsOptions` or similar.